mock-thunderhead/src/main/resources/application.yml

```diff
@@ -57,7 +57,7 @@ auth:
     datahub:
       runtime.upgrade.enable: true
       os.upgrade.enable: true
-      instancetypes.enable: false
+      instancetypes.enable: true
```

Comment:
Note: I do not mind leaving this as long as it has no negative effect on E2E test. (I can see the mock IT succeeded on the PR, which is good, but cloud platform specific stuff may still depend on this entitlement.) Anyway, let us revisit this later if needed.